ui/perfherder/alerts/FileBugModal.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Form,
+  FormGroup,
+  Input,
+  Label,
+  Button,
+  Modal,
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+  Col,
+  Row,
+} from 'reactstrap';
+import debounce from 'lodash/debounce';
+
+export default class FileBugModal extends React.Component {
+  // eslint-disable-next-line react/sort-comp
+  constructor(props) {
+    super(props);
+    this.state = {
+      inputValue: '',
+      invalidInput: false,
+      validated: false,
+    };
+  }
+
+  validateInput = debounce(() => {
+    const { inputValue } = this.state;
+    const regex = /^[1-9]+[0-9]*$/;
+    const updates = { validated: false, invalidInput: false };
+
+    if (!inputValue.length) {
+      updates.validated = true;
+      updates.invalidInput = false;
+    }
+
+    if (!inputValue.match(regex)) {
+      updates.invalidInput = true;
+    }
+
+    this.setState(updates);
+  }, 0);
+
+  updateInput = (event) => {
+    this.setState(
+      { inputValue: event.target.value, validated: false },
+      this.validateInput,
+    );
+  };
+
+  handleSubmit = async (event, inputValue) => {
+    const { updateAndClose } = this.props;
+    await updateAndClose(event, inputValue);
+    this.setState({ inputValue: '' });
+  };
+
+  render() {
+    const {
+      showModal,
+      toggle,
+      dropdownOption,
+      header,
+      title,
+      submitButtonText,
+    } = this.props;
+
+    const { inputValue, invalidInput, validated } = this.state;
+
+    return (
+      <Modal isOpen={showModal}>
+        <ModalHeader toggle={toggle}>{header}</ModalHeader>
+        <Form>
+          <ModalBody>
+            <FormGroup>
+              <Row className="justify-content-center">
+                <Col className="col-4">
+                  <Label for="taskId">{title}</Label>
```

Comment:
The label could be renamed to culprit id.

Reply:
This is how it was before, but I thought at a more "objective" name like `Submit bug for`.